plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/Circle.java

```diff
@@ -35,12 +42,23 @@
    * To update the circle on the map use {@link CircleManager#update(Annotation)}.
    * <p>
    *
-   * @param latLng the location of the circle in a longitude and latitude pair
+   * @param latLng the location of the circle in a latitude and longitude pair
    */
   public void setLatLng(LatLng latLng) {
     geometry = Point.fromLngLat(latLng.getLongitude(), latLng.getLatitude());
   }
 
+  /**
+   * Get the LatLng of the circle, which represents the location of the circle on the map
+   *
+   * @return the location of the circle
+   */
+  @NonNull
+  public LatLng getLatLng() {
+    Point point = (Point) geometry;
```

Comment:
not related to this PR, but something that comes to mind now, do we need type geometry with generics? (avoids casting in this case).

Reply:
we can possibly do it to clean up those casts :+1: